api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java

```diff
@@ -52,52 +61,69 @@ public EgressEventServiceImpl(
+      WorkspaceDao workspaceDao) {
     this.clock = clock;
     this.egressEventAuditor = egressEventAuditor;
     this.institutionService = institutionService;
     this.alertApiProvider = alertApiProvider;
     this.workbenchConfigProvider = workbenchConfigProvider;
     this.userService = userService;
     this.workspaceAdminService = workspaceAdminService;
+    this.workspaceDao = workspaceDao;
   }
 
   @Override
   public void handleEvent(EgressEvent event) {
+    // Lookup workspace by googleProject name, and set the workspaceNamespace in EgressEvent.
+    Optional<DbWorkspace> dbWorkspaceMaybe =
+        workspaceDao.getByGoogleProject(event.getProjectName());
+    String workspaceNamespace;
+    if (!dbWorkspaceMaybe.isPresent()) {
+      logger.warning(
+          String.format(
+              "Workspace not found by given Google Project Id: %s", event.getProjectName()));
+      workspaceNamespace = NOT_FOUND_WORKSPACE_NAMESPACE;
+    } else {
+      workspaceNamespace = dbWorkspaceMaybe.get().getWorkspaceNamespace();
+    }
+
     logger.warning(
         String.format(
-            "Received an egress event from project %s (%.2fMiB, VM prefix %s)",
-            event.getProjectName(), event.getEgressMib(), event.getVmPrefix()));
+            "Received an egress event from workspace namespace %s, googleProject %s (%.2fMiB, VM prefix %s)",
+            workspaceNamespace, event.getProjectName(), event.getEgressMib(), event.getVmPrefix()));
     this.egressEventAuditor.fireEgressEvent(event);
-    this.createEgressEventAlert(event);
+    this.createEgressEventAlert(event, workspaceNamespace);
   }
 
   // Create (or potentially update) an OpsGenie alert for an egress event.
   private SuccessResponse createAlert(CreateAlertRequest createAlertRequest) throws ApiException {
     return this.alertApiProvider.get().createAlert(createAlertRequest);
   }
 
-  private void createEgressEventAlert(EgressEvent egressEvent) {
-    final CreateAlertRequest createAlertRequest = egressEventToOpsGenieAlert(egressEvent);
+  private void createEgressEventAlert(EgressEvent egressEvent, String workspaceNamespace) {
+    final CreateAlertRequest createAlertRequest =
+        egressEventToOpsGenieAlert(egressEvent, workspaceNamespace);
     try {
       final SuccessResponse response = createAlert(createAlertRequest);
       logger.info(
           String.format(
-              "Successfully created or updated Opsgenie alert for high-egress event on project %s (Opsgenie request ID %s)",
-              egressEvent.getProjectName(), response.getRequestId()));
+              "Successfully created or updated Opsgenie alert for high-egress event on project %s, namespace %s (Opsgenie request ID %s)",
+              egressEvent.getProjectName(), workspaceNamespace, response.getRequestId()));
     } catch (ApiException e) {
       logger.severe(
           String.format(
               "Error creating Opsgenie alert for egress event on project %s: %s",
```

Comment:
This says "project" and then displays the namespace.  Please change to something like "Google project %s, namespace %s"

Reply:
added. PTAL. Thanks